src/Controls/tests/UITests/Tests/Concepts/ImageLoadingGalleryTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Maui.Controls.Sample;
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests
+{
+	public class ImageLoadingGalleryTests : CoreGalleryBasePageTest
+	{
+		public ImageLoadingGalleryTests(TestDevice device)
+			: base(device)
+		{
+		}
+
+		protected override void NavigateToGallery()
+		{
+			App.NavigateToGallery("Image Loading Gallery");
+		}
+
+		[Test]
+		public void LoadAndVerifyPng() => LoadAndVerify(Test.ImageLoading.FromBundlePng);
```

Comment:
Nice!. Need to include the Windows snapshots.
![image](https://github.com/dotnet/maui/assets/6755973/4da92c0d-4d26-4826-a290-51f8c8dd5bd1)

Reply:
that was windows just blowing up for some reason, rerunning in hopes and dreams